### ISSUE TITLE
Timestamp parsing ignores milliseconds

### ISSUE THREAD

**Issue by Schabernack** [kilianc/rtail#89]
The following line:

```
[2015-08-05 01:47:32,369] [6732] [api] [DEBUG] {before_request:70} ##### NEW REQUEST #####
```

Shows up in rtail like this:

```
08/04/15 09:47:32
369] [6732] [api] [DEBUG] {before_request:70} ##### NEW REQUEST #####
```

The First line is the timestamp that is detected by rtail. The second line is the log as shown by rtail: `369]` should either be discarded or preferably be part of the timestamp.

**Comment by kilianc** [comment]
Yes I think rtail needs to replace the `chrono` library with something else. For the time being please disable the date parsing

**Comment by Schabernack** [comment]
Thanks. Disabling date parsing also helps with Issue #88 .
